free-1.md

```diff
@@ -10,41 +10,157 @@ _What need is met by your idea? Who are you helping? What is that person's
 experience like now? What would their experience be like if you could help 
 them?_
 
+Tableau trees is widely used in logic. 
+There is a LaTeX 
+[package that's specifically for drawing tableau trees](http://ctan.org/pkg/prooftrees)
+but its syntax is quite challenging.
+So this project would be for logicians who are not familiar enough with LaTeX
+to feel comfortable using the packages for drawing trees.
+Ideally, this project would eliminate the need to drawing out lines and moving
+text boxes using formatting tools like Google Slides, which is the recommended
+formatting tool in CS 81.
```

Comment:
This sounds like a very useful idea (I wish I had it CS 81..). I like the idea of avoiding LaTex because it has a very unnatural syntax and Google Sheets/Slides/Docs was always hard to make proper trees. I am curious to learn more about this.